addons/medical_damage/stringtable.xml

```diff
@@ -15,15 +15,15 @@
             <German>Schwelle für kritischen Spielerschaden</German>
         </Key>
         <Key ID="STR_ACE_Medical_Damage_PlayerDamageThreshold_Description">
-            <English>Sets the amount of damage a player can receive before going unconscious.</English>
+            <English>Sets the amount of damage a player can receive before going unconscious or dying if "Sum of Trauma" is enabled.</English>
             <Japanese>プレイヤーが気絶になる前に受けられるダメージ量を決定できます。</Japanese>
             <French>Définit la quantité de dégâts qu'un joueur peut subir avant de perdre connaissance.</French>
             <Russian>Устанавливает количество урона, которое может получить игрок, прежде чем потеряет сознание.</Russian>
             <Portuguese>Define a quantidade de dano que um jogador pode receber antes de ficar inconsciente.</Portuguese>
             <Chinese>設定玩家在無意識前能承受多少傷害。</Chinese>
             <Italian>Imposta la quantità di danno che un giocatore può ricevere prima di perdere conoscenza.</Italian>
             <Czech>Nastavuje kolik poškození hráč může obdržet než upadne do bezvědomí.</Czech>
-            <Polish>Definiuje ilość obrażeń jaką może przyjąć gracz przed straceniem przytomności.</Polish>
+            <Polish>Definiuje ilość obrażeń jaką może przyjąć gracz przed straceniem przytomności oraz śmierci gdy "Suma urazów" jest włączona.</Polish>
             <Spanish>Fijar la cantidad de daño que un jugador puede recivir antes de caer inconsciente</Spanish>
             <German>Legt die Höhe des Schadens fest, den ein Spieler erhalten kann, bevor er ohnmächtig wird.</German>
```

Comment:
```suggestion
            <German>Legt die Höhe des Schadens fest, den ein Spieler erhalten kann, bevor er ohnmächtig wird (oder stirbt, wenn "Summe aller Traumata" aktiviert ist).</German>
```

Reply:
What's up with GitHub showing XML diffs like 1 space is removed?

Reply:
I'll add the parenthesis to polish and english too. Makes the description more clear.